layouts/_default/baseof.html

```diff
@@ -54,14 +54,7 @@
   <meta name="google-site-verification" content="sBa9o-9RVlIHHlyeaHgIir20ZKcSuzLVgCCeXIMgKZg" />
 </head>
 
-{{ with $pageTitle }}
-
-<body vocab="http://schema.org/" typeof="WebPage" class="{{ $pageTitle }} {{ (string ( $.Site.Language) ) }}">
-  {{ else }}
-
-  <body vocab="http://schema.org/" typeof="WebPage"
-    class="{{if eq .Section "blog"}}page-blogpost{{else}}{{ lower .Params.title }}{{end}} {{ (string ( $.Site.Language) ) }}">
```

Comment:
Note that the `page-blogpost` class gets used in a few places,
<img width="296" alt="Screen Shot 2020-12-02 at 11 00 38" src="https://user-images.githubusercontent.com/263472/100912469-bc5a4e00-348d-11eb-9c4d-5eca19d5a86c.png">